ml-proto/spec/kernel.ml

```diff
@@ -119,16 +119,18 @@ and func' =
 type memory = memory' Source.phrase
 and memory' =
 {
-  initial : Memory.size;
+  min : Memory.size;
```

Comment:
Ah, I hadn't noticed that BinaryEncoding.md calls it 'min' too; filed a [PR](https://github.com/WebAssembly/design/pull/661) to update BinaryEncoding.md.  ([Modules.md](https://github.com/WebAssembly/design/blob/master/Modules.md#linear-memory-section) already calls it initial.)